content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -196,7 +233,73 @@ System.out.println(set.higher(23));  // ERROR, no such element exists
 ```
 
 </JavaSection>
+<PySection>
+
+Python does not have a sorted set, so see the C++ or Java if you want an
+implementation from the standard library.
+
+Creating a sorted set from scratch is time consuming, and not reccomended to do
+so in contests, but if you are curious regarding the implementation, you can
+find an AVL tree implementation of a sorted set below.
+
+<Resources>
+	<Resource source="DSA Python" url="https://nibmehub.com/opac-service/pdf/read/Data%20Structures%20and%20Algorithms%20in%20Python.pdf#page=503" title="AVL Trees">
+		definition and implementation of AVL Trees in Python
+	</Resource>
+</Resources>
+
+<Warning>
+
+You are not expected to know how to create an AVL tree in USACO.
+
+</Warning>
+
+Since some online judges such as LeetCode include additional libraries, there is
+also an implementation of sorted sets included from the `sortedcontainers`
+library. All operations shown below are $\mathcal{O}(\log N)$ time, except for
+its $\mathcal{O}(N \log N)$ initialization.
```

Comment:
you already use sortedset before mentioning this
perhaps just remove the sortedset codesnip above